web/src/components/pages/guidelines/sidebar-content/voice-collection/misreadings.tsx

```diff
@@ -0,0 +1,87 @@
+import { Localized } from '@fluent/react';
+import classNames from 'classnames';
+import * as React from 'react';
+
+import { ChevronDown } from '../../../../ui/icons';
+import { TextButton } from '../../../../ui/ui';
+import { SidebarContentProps } from '../../types';
+
+import { ExampleContent, ExampleTip } from './example-content-voice';
```

Comment:
I couldn't figure out where will live the actual guidelines (per language) and whether we'll be able to amend them on a per language basis.

Reply:
ping